### ISSUE TITLE
Units.LogarithmicDecrement has units of Ohms (1/S).

### ISSUE THREAD

**Issue by JoeRiel** [modelica/ModelicaStandardLibrary#3853]
Possibly the intention was 1/s, but even that doesn't make sense.  It should be dimensionless.

**Comment by HansOlsson** [comment]
Assuming it is the same as https://en.wikipedia.org/wiki/Logarithmic_decrement I agree that it should be dimensionless. As far I can see it is not documented and not used in MSL so I cannot be certain that I interpret it correctly.

**Comment by christiankral** [comment]
I agree with @JoeRiel: `Units.LogarithmicDecrement` shall be dimensionless.
